src/components/Sketches/index.js

```diff
@@ -169,13 +169,19 @@ class Sketches extends React.Component {
       onClose={() => this.setConfirmDeleteModalOpen(false)}
       sketchName={this.state.selectedSketch}
       sketchKey={this.state.selectedKey}
+      pid={
+        Array.from(this.props.programs)[this.state.selectedKey]
+          ? Array.from(this.props.programs)[this.state.selectedKey].uid
+          : ""
+      }
```

Comment:
I'm not a huge fan of how I made the changes to this file, but these felt the most natural without modifying much else. Thoughts?